tests/test_photonphase.py

```diff
@@ -4,7 +4,7 @@
 
 import pytest
 
-import pint.scripts.photonphase as photonphase
+import photonphase
```

Comment:
I'm wondering why this change was needed?

Reply:
Actually, it seems to be causing test failures. I think this needs to be reverted.